browser/src/RegionalMissenseConstraintTrack.tsx

```diff
@@ -216,7 +214,7 @@ const RegionTooltip = ({ region, isTranscriptWide }: RegionTooltipProps) => {
     <RegionAttributeList>
       <div>
         <dt>Coordinates:</dt>
-        <dd>{`${region.chrom}:${region.region_start}-${region.region_stop}`}</dd>
+        <dd>{`${region.chrom}:${region.start}-${region.stop}`}</dd>
```

Comment:
`region_start` and `region_stop` while in hindsight, probably poorly named, are not redundant. Also in hindsight this may have been a good place for a comment in the code discussing this, since it was a specific request and a bit subtle.

The track renders and colors each exon or portion of an exon based on the `start` and `stop` coordinates output from finding the overlap of the two passed in region arrays, but the RMC team wanted the hover over text to reflect the coordinates of the constrained region (`region_start` and `region_stop`), not the coordinates of the exon or portion of the exon (`start` and `stop`). 

The RMC team had reached out to us to change the behavior to be this way, and at that time passing the `region_start` and `region_stop` coordinates so that each output overlapping region would have those values was my solution.

---
Screenshots

In production, since these two exons are part of the same constrained region, the tooltips show the start and stop coordinates for the region:
- ![Screenshot 2024-05-02 at 09 18 04](https://github.com/broadinstitute/gnomad-browser/assets/59549713/cabcf0a0-d539-4b49-9bd8-4cca930e6df7)
-![Screenshot 2024-05-02 at 09 18 08](https://github.com/broadinstitute/gnomad-browser/assets/59549713/1ae5ef74-614e-4f85-8d92-c8074bd790d5)

With this change, this behavior has reverted to displaying the coordinates for the block thats rendered:
- ![Screenshot 2024-05-02 at 09 17 22](https://github.com/broadinstitute/gnomad-browser/assets/59549713/17e0000d-3f8c-4cfc-b69e-8a00477300a8)
- ![Screenshot 2024-05-02 at 09 17 25](https://github.com/broadinstitute/gnomad-browser/assets/59549713/0249f0dc-ca2a-4635-88b0-8a3c235ba628)